core/templates/pages/topics-and-skills-dashboard-page/navbar/topics-and-skills-dashboard-navbar.directive.ts

```diff
@@ -135,6 +138,7 @@ angular.module('oppia').directive('topicsAndSkillsDashboardNavbar', [
                 }
               ]
             }).result.then(function(result) {
+              ContextService.unsetSaveImagesToLocalStorageContext();
```

Comment:
@aks681, the general structure for the frontend changes LGTM! I'm not sure how we will handle the case while editing the skill/question? SO will it directly upload images in edit mode?

Reply:
Yeah, when editing, it will directly upload. 
Basically, whenever this flag isn't set, it will directly upload. So, if we set this flag in the editor and manage it properly when 'Save Changes' is clicked (haven't done that for POST yet either), we can have the same function when editing as well, though it isn't required for the skill editor.

Reply:
Ah, I see. Thanks for the info!